sqldelight-studio-plugin/src/main/kotlin/com/squareup/sqldelight/lang/SqliteFile.kt

```diff
@@ -19,12 +19,11 @@ import com.intellij.extapi.psi.PsiFileBase
 import com.intellij.psi.FileViewProvider
 import com.intellij.psi.PsiFile
 import com.squareup.sqldelight.Status
-import kotlin.properties.Delegates.notNull
 
 class SqliteFile internal constructor(viewProvider: FileViewProvider)
 : PsiFileBase(viewProvider, SqliteLanguage.INSTANCE) {
   var generatedFile: PsiFile? = null
-  var status: Status by notNull()
+  var status: Status? = null
```

Comment:
in big projects it looks like theres a race condition with the annotator and the generator. I'll look into it more later and follow up.